src/core/public/chrome/ui/header/branding/opensearch_dashboards_custom_logo.tsx

```diff
@@ -32,13 +32,16 @@
 
 import React from 'react';
 import '../header_logo.scss';
+import { OpenSearchDashboardsLogoDarkMode } from './opensearch_dashboards_logo_darkmode';
 
 export interface CustomLogoType {
-  logoUrl: string;
+  logoUrl: string | undefined;
 }
 
 export const CustomLogo = ({ ...branding }: CustomLogoType) => {
```

Comment:
can we add some jsdoc with the @param defined

Reply:
Added jsdoc with @params defined.

Reply:
apologies meant the CustomLogo, like defining `branding` and what it returns and brief description.